modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java

```diff
@@ -99,6 +100,9 @@ public class CodeGenMojo extends AbstractMojo {
     @Parameter(name = "output", property = "openapi.generator.maven.plugin.output")
     private File output;
 
+    @Parameter(name = "cleanUpOutputFolder", property = "openapi.generator.maven.plugin.cleanUpOutputFolder", defaultValue = "false")
```

Comment:
what about naming the option as `cleanupOutput` to make it consistent with the gradle plugin?

ref: https://github.com/OpenAPITools/openapi-generator/pull/13659

Reply:
@wing328 I've changed name